docs/src/tutorial/cython_tutorial.rst

```diff
@@ -325,7 +405,7 @@ Primes with C++
 With Cython, it is also possible to take advantage of the C++ language, notably,
 part of the C++ standard library is directly importable from Cython code.
 
-Let's see what our :file:`primes.pyx` becomes when
+Let's see what our code becomes when
 using `vector <https://en.cppreference.com/w/cpp/container/vector>`_ from the C++
 standard library.
```

Comment:
```suggestion
Let's see what our code becomes when using
`vector <https://en.cppreference.com/w/cpp/container/vector>`_ from the C++
standard library.
```

Reply:
Nothing changes from the point of view of the reader of the documentation. If you wanted to make it easier to change and view the source rst, then you also had to wrap the end of the line, i.e. "from the C ++".

Reply:
Yes, I think it helps the maintenance to format it that way.

Reply:
resolved [here](https://github.com/cython/cython/pull/4226/commits/74e39646d3a0a006c16320ef428d26d0c4fcc147)